lib/suspenders/app_builder.rb

```diff
@@ -322,6 +318,7 @@ def gitignore_files
       copy_file 'suspenders_gitignore', '.gitignore'
       [
         'app/views/pages',
+        'app/workers',
```

Comment:
It creates a `.keep` file in `app/workers` so that empty dir exists. The method naming is really poor here (`gitignore_files` — wat?)